components/layout_2020/formatting_contexts.rs

```diff
@@ -125,6 +127,14 @@ impl IndependentFormattingContext {
                             ),
                         )
                     },
+                    DisplayInside::Grid => {
+                        NonReplacedFormattingContextContents::Grid(TaffyContainer::construct(
+                            context,
+                            node_and_style_info,
+                            non_replaced_contents,
+                            propagated_text_decoration_line,
+                        ))
+                    },
```

Comment:
Shouldn't this be dependent on the new preference?

Reply:
This codepath can't be reached with the preference disabled anyway, because Stylo will also have the grid preference disabled which means that it won't parse `display: grid`, and you'll get whatever the fallback `display` value is. I can put in an `unreachable!()` fallback branch if you want?

Reply:
That's not necessary. I just didn't understand how this was being disabled.